test/index.js

```diff
@@ -19,16 +31,28 @@ describe('switchPath', () => {
         '/foo': 456,
       },
     });
+
     expect(path).to.be.equal('/home/foo');
     expect(value).to.be.equal(456);
   });
 
-  it('should match a path on an incomplete pattern', () => {
+  it('should NOT match a path on an incomplete pattern', () => {
     const {path, value} = switchPath('/home/foo', {
       '/bar': 123,
       '/home': 456,
     });
-    expect(path).to.be.equal('/home');
+    expect(path).to.be.equal(null);
+    expect(value).to.be.equal(null);
+  });
```

Comment:
But I'd like to merge the other features. Can you somehow revert changes just for this test to behave like previously?